syntax = "proto3";
package helloworld;

// 请求与回复消息
message Point3{
    float x = 1;
    float y = 2;
    float z = 3;
}

message Numeric{
    float value = 1;
}
// 简单打招呼服务
service ProcessingServices {
  rpc computeSum(Point3) returns (Numeric){}
}
